apps/meteor/lib/phoneValidator.ts

```diff
@@ -0,0 +1,6 @@
+export const validatePhone = (phone: string): boolean => {
+    const regex = /^[+]?[\s./0-9]*[(]?[0-9]{1,4}[)]?[-\s./0-9]*$/g;
```

Comment:
Instead of hardcoding this, I'd ask you to make this configurable by adding a setting for it on the admin panel. This way if some user is trying to enter a number which is valid for them, then they can modify the regex to suite their needs